python/pytests/golden/hash_test/test_hash_string.jsonl

```diff
@@ -0,0 +1,6 @@
+{"_time":"2021-01-01T00:00:00.000000000","_key":"Ben","m":"hello","hash_m":1472103086483932002}
+{"_time":"2021-01-01T00:00:00.000000000","_key":"Ryan","m":null,"hash_m":0}
+{"_time":"2021-01-02T00:00:00.000000000","_key":"Ryan","m":"world","hash_m":8057155968893317866}
+{"_time":"2021-01-03T00:00:00.000000000","_key":"Ben","m":"hi","hash_m":2460612554838835252}
+{"_time":"2021-01-04T00:00:00.000000000","_key":"Ben","m":null,"hash_m":0}
+{"_time":"2021-01-04T00:00:00.000000000","_key":"Ryan","m":"earth","hash_m":14489671231712828724}
```

Comment:
these values aren't what I expected... old docs show different values:
 https://kaskada.io/docs-site/kaskada/main/fenl/catalog.html#hash

also I wasn't expecting `null` to hash to `0`